### PR TITLE
[ToricVarieties] Method blow_up returns the blowdown morphism

### PR DESCRIPTION
**Pull request by HereAround**
As request by @HechtiDerLachs quite a long time ago.

cc @lkastner

### CODE REVIEW

File: src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/standard_constructions.jl [hunk 1]
```diff
@@ -410,25 +404,34 @@ Multivariate polynomial ring in 5 variables over QQ graded by
...
+      new_vars[i] = j !== nothing ? old_vars[j] : coordinate_name
+  end
+  if set_attributes
+    set_attribute!(new_variety, :coordinate_names, new_vars)
+  end
+  dim = ambient_dim(fan(v))
+  m = zero_matrix(ZZ, dim, dim)
+  for i in 1:dim
+    m[i, i] = one(ZZ)  
+  end
```
Comment (afkafkafk13) [inline src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/standard_constructions.jl]: Isn't there a predefined way to obtain the unit-matrix of size `dim` available?

**Comment by afkafkafk13** [comment]
Thank you for this compatibility change. It is in general easier to obtain the blow-up scheme from the blow-down morphism than the other way around.
Apart from the comment above, this change looks good to me.

File: src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/standard_constructions.jl (hunk 1 above)
Comment (HereAround) [reply src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/standard_constructions.jl]: Yes, `identity_matrix(ZZ, dim)`, as defined in `AbstractAlgebra.jl`. Thank you for reminding me. Adjusted.

**Review by afkafkafk13 — APPROVED** [review]

**Comment by afkafkafk13** [comment]
Let us wait, what @lkastner responds to the review request from his perspective before merging.

**Review by lkastner — APPROVED** [review]

**Comment by HereAround** [comment]
@afkafkafk13 In the run https://github.com/oscar-system/Oscar.jl/actions/runs/5782481338/job/15669381307?pr=2616, there is an error from the du Val tester:

```julia
du Val Tester: Error During Test at /home/runner/work/Oscar.jl/Oscar.jl/test/AlgebraicGeometry/Schemes/duValSing.jl:16
  Test threw exception
  Expression: has_du_val_singularities(X) == false
  Ideal does not describe a single K-point
```

Please advice on how to proceed.

**Comment by afkafkafk13** [comment]
> @afkafkafk13 In the run https://github.com/oscar-system/Oscar.jl/actions/runs/5782481338/job/15669381307?pr=2616, there is an error from the du Val tester:
> 
> ```julia
> du Val Tester: Error During Test at /home/runner/work/Oscar.jl/Oscar.jl/test/AlgebraicGeometry/Schemes/duValSing.jl:16
>   Test threw exception
>   Expression: has_du_val_singularities(X) == false
>   Ideal does not describe a single K-point
> ```
> 
> Please advice on how to proceed.

My advice here: 
Rerun failed tests. This is an error from a non-caught corner case from absolute primary decomposition/absolute factorization. As absolute primary decomposition/absolute factorization are using random  and as this corner case has a low (but not extremely low) probability, the error will most likely go away in the second run or the third at the latest. (@wdecker and Hans are working on it.)

**Comment by HereAround** [comment]
> > @afkafkafk13 In the run https://github.com/oscar-system/Oscar.jl/actions/runs/5782481338/job/15669381307?pr=2616, there is an error from the du Val tester:
> > ```julia
> > du Val Tester: Error During Test at /home/runner/work/Oscar.jl/Oscar.jl/test/AlgebraicGeometry/Schemes/duValSing.jl:16
> >   Test threw exception
> >   Expression: has_du_val_singularities(X) == false
> >   Ideal does not describe a single K-point
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Please advice on how to proceed.
> 
> My advice here: Rerun failed tests. This is an error from a non-caught corner case from absolute primary decomposition/absolute factorization. As absolute primary decomposition/absolute factorization are using random and as this corner case has a low (but not extremely low) probability, the error will most likely go away in the second run or the third at the latest. (@wdecker and Hans are working on it.)

Thank you. Worked!
